components/automate-ui/src/app/modules/infra-proxy/chef-server-details/chef-server-details.component.html

```diff
@@ -91,9 +91,9 @@
           <div id="button-bar">
             <chef-button [disabled]="isLoading || !updateServerForm.valid || !updateServerForm.dirty"
               primary inline (click)="saveServer()">                
-            <chef-loading-spinner *ngIf="saving"></chef-loading-spinner>
-            <span *ngIf="saving">Saving...</span>
-            <span *ngIf="!saving">Save</span></chef-button>
+            <chef-loading-spinner *ngIf="saveInProgress"></chef-loading-spinner>
+            <span *ngIf="saveInProgress">Saving...</span>
+            <span *ngIf="!saveInProgress">Save</span></chef-button>
             <span id="saved-note" *ngIf="saveSuccessful && !updateServerForm.dirty">All changes saved.</span>
           </div>
         </form>
```

Comment:
Technically beyond the scope of this PR, but since you are in here would appreciate if you could make this fix as well.

Background: The `formControlName`, besides its normal function to Angular, is also used by our `FormControlDirective` to handle enabling/disabling the <kbd>Save</kbd> button based on a field's value.

Here is the scenario:
Go to Infrastructure >> Chef Servers >> (server) >> Details.
Assume the server name was just "name". Change it to "name1" and <kbd>Save</kbd> becomes enabled. Change it back to "name" and <kbd>Save</kbd> becomes disabled. That magic is done by  `FormControlDirective`, and that is working on your form.

The missing part: Change the value to "name1" (just a different value from the original) so you can press <kbd>Save</kbd>. Go ahead and press <kbd>Save</kbd>. 
Upon completion <kbd>Save</kbd> becomes disabled. ✅ 
Change the value to, e.g. "name2". <kbd>Save</kbd> becomes enabled. ✅ 
Change the value back to "name1". <kbd>Save</kbd> should become disabled. ❌ 

In order to fix that last step, you need to add a `resetOrigin` to each `<input>` containing a `formControlName`. Example:
```
<input chefInput formControlName="description" [resetOrigin]="saveSuccessful" type="text" autocomplete="off"
```
For details of why/how this works, see the documentation in components/automate-ui/src/app/components/form-control/form-control.directive.ts

Reply:
thanks for pointing it out :1st_place_medal: